zio-http/src/main/scala/zio/http/Driver.scala

```diff
@@ -2,10 +2,35 @@ package zio.http
 
 import zio._
 
+import java.util.concurrent.atomic.AtomicReference
+
 trait Driver {
   def start: RIO[Scope, Int]
 
-  def setErrorCallback(newCallback: Option[Server.ErrorCallback]): UIO[Unit]
+  def setErrorCallback(newCallback: Option[Server.ErrorCallback]): Unit
+
+  def addApp(newApp: HttpApp[Any, Throwable]): Unit
+}
+
+object Driver {
+
+  private[zio] final case class Context private (
```

Comment:
This class (`Driver.Context`) should be regarded as a private implementation detail for _some_ drivers, but not others, and moved to some place Netty-specific.

Reply:
It actually shouldn't exist at all.  It was replaced by new methods in the Driver.